### ISSUE TITLE
[Shadowrun 5E] Some broken whisper toggles and grunt skill checks

### ISSUE THREAD

**Issue by Orudeon** [Roll20/roll20-character-sheets#5511]
On the grunt character sheet, at least the following whisper toggles are broken:
- Data Processing
- Attack
- Sleaze
- Firewall

I haven't found any others at the time of writing this.

At least the following skill rolls are broken:
- Negotiation
- Archery
- Pistols
- Firearms Group
- Unarmed Combat
- Athletics Group

I'm unable to exhaustively test every skill right now, those are the ones I found.

Also, if/when the character sheet update to 2.0 is pushed, does it retroactively apply to my game? If not, how would I update it?

**Comment by clevett** [comment]
Which character sheet is this for? Shadowrun 5th Edition or Shadowru 5e Advanced?

**Comment by Orudeon** [comment]
It looks like the bottom of the sheet says it's 2.0 in tiny text.

Also, my players are reporting that they can't roll anything but they're still filling out things. Is there something in particular that breaks that? My sheet rolls fine. I could invite a developer to look at them?

**Comment by Orudeon** [comment]
Also, unchecking Modifiers results in the following error:
SyntaxError: Expected "(", ".", "[", "abs(", "ceil(", "d", "floor(", "round(", "t", "{", [ |\t], [+|\-], [+|\-|*|\/|%] or [0-9] but ")" found.

**Comment by Orudeon** [comment]
I may as well just dump a bunch of feedback here because I don't see the point in making a new thread for every single one.

From an accessibility standpoint, my colorblindness makes it impossible for me to see when edge and matrix condition monitors are depleted -- or not. I'd personally appreciate a substantially different color, this dark red and black isn't working for me. You can use https://webaim.org/resources/contrastchecker/ to manually play around with colors to make maximizing the ease of access for your users very simple.

**Comment by clevett** [comment]
This was certainly a project I did before learning Accessibility programming. I'll look at adding higher contrast or a better visual indicator for Edge/Matrix Damage.

That Modifier error is really odd. I'm not able to replicate it. Could you go to Attributes & Abilities tab on the Roll20 window then try deleting modifier_toggle ? Toggle Modifier on and off again. See if that helps.

You can find me on [Roll20](https://app.roll20.net/users/557736/cassie) or Discord (Nacoya #7621) and send me a game invite. I'll try to take a look of what it might be.

**Comment by Orudeon** [comment]
My Discord is Orudeon#4767, I don't know why the requests are being bounced by discord. I have this problem all the time. I also sent you a DM on roll20.

Also while fiddling with something else I discovered that pressing the armor rating number (the 12 in the attached)
![chrome_2019-10-01_17-24-57](https://user-images.githubusercontent.com/16169101/66009689-70b6a980-e470-11e9-89b0-e1c780195650.png)
 under Core Combat Info prints "No attribute was found for @{CHARACTER NAME|armor}"

I fixed it by manually adding an armor attribute and putting the name of the armor in the field
